Mathlib/Algebra/GroupPower/Order.lean

```diff
@@ -338,7 +328,7 @@ end CovariantLTSwap
 @[to_additive Left.nsmul_neg_iff]
 theorem Left.pow_lt_one_iff' [CovariantClass M M (· * ·) (· < ·)] {n : ℕ} {x : M} (hn : 0 < n) :
     x ^ n < 1 ↔ x < 1 :=
-  haveI := Mul.to_covariantClass_left M
+  haveI := covariantClass_le_of_lt M M (· * ·)
```

Comment:
I'm not in favor of the underscore, but if it's the convention then so be it!

Reply:
Actually, since `haveI` was already in the original code let me not change it; we could do a global find and replace sometime.

Reply:
I'm happy to leave as is.  [This comment on Zulip](https://leanprover.zulipchat.com/#narrow/stream/116395-maths/topic/.E2.9C.94.20OfNat.2C.20Subtypes.20issues.20for.20ConvexCone.2EPointed/near/379317740) mentions the `haveI` vs `let` question.  Still, I am not sure whether the reason for preferring `let` is simply "because now it works".